strax/corrections.py

```diff
@@ -0,0 +1,120 @@
+'''I/O format for corrections using MongoDB
+
+This module list, reads, writes, among others functionalities
+to a MongoDB where corrections are stored.
+'''
+
+from datetime import datetime
+import logging
+import pandas as pd
+import pdmongo as pdm
+import pymongo
+
+import strax
+
+export, __all__ = strax.exporter()
+
+
+@export
+class CorrectionsInterface:
+    '''A class to manage corrections that are stored in a MongoDB.
+    '''
+
+    def __init__(self, host='127.0 0.1', username=None, password=None,
+                 database_name='corrections'):
+
+        self.host = host
+        self.username = username
+        self.password = password
+        self.client = pymongo.MongoClient(host=self.host,
+                                          username=self.username,
+                                          password=self.password)
+        self.database_name = database_name
+        self.database = self.client[self.database_name]
+
+    def list_corrections(self):
+        '''Smart logic to list corrections.
+        '''
+        return [x['name'] for x in self.database.list_collections()
+                if 'global' not in x['name']]
```

Comment:
Maybe make global a keyword argument. Or will this be the name always "global" in a MongoDB?

Reply:
As I mentioned previously global is a reference to a set of corrections (dataframes), I can list all corrections available including global

Reply:
So with other words, it could in principle change? Then I would suggest to make it a keyword argument, does not hurt.

Reply:
As the function says it list *all* corrections, no keyword argument needed